mern/client/src/DAO/recoverSessionDAO.js

```diff
@@ -0,0 +1,15 @@
+import axios from 'axios';
+// DAO is the inner most layer because DB stores our entities (flow, QA, transcript)
+
+// Returns flow json object with matching sessionID
+export async function getFlowBySessionID(sessionID) {
+    try {
+      const response = await axios.get(`http://localhost:5000/flow/${sessionID}`, sessionID);
+      const { data } = response;
+      // return the flow object
+      return data;
+
+    } catch (e) {
+      return false;
```

Comment:
would this return anything to the user or is this return only for the function?

Reply:
This returns the flow object from the DB up to the interactor, then the controller, where the QA's are extracted and displayed on the screen for the user.

Reply:
I see thank you!